tests/functional/object/mcg/test_noobaa_secret.py

```diff
@@ -355,3 +363,32 @@ def test_noobaa_root_secret():
         check_if_mcg_root_secret_public() is False
     ), "Seems like MCG root secrets are exposed publicly, please check"
     logger.info("MCG root secrets are not exposed to public")
+
+
+@mcg
+@red_squad
+@acceptance
+@bugzilla()
+@polarion_id()
+def test_operator_logs_for_secret():
```

Comment:
can you please cover https://bugzilla.redhat.com/show_bug.cgi?id=2240778 if possible in this PR?

Reply:
yes I will check

Reply:
test is added